src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/Configuration/SelectAttributeConfigurationType.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Class SelectAttributeConfigurationType
+ * @package Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration
+ */
+class SelectAttributeConfigurationType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('options', 'collection', [
+                'type' => 'text',
```

Comment:
Just an idea, if you could somehow ask for a `type` (*text* or *color*) configuration, we can have color inputs and collection of different colors (instead of string color names `Red, `Blue`, etc)

Currently modern browsers (except IE) which store admins normally use, does support `color` input type.